xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-ui/src/main/resources/CKEditor/ContentSheet.xml

```diff
@@ -59,7 +59,7 @@
   #else
     &lt;!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"&gt;
   #end
-  &lt;html xmlns="http://www.w3.org/1999/xhtml" lang="$xcontext.locale" xml:lang="$xcontext.locale"&gt;
+  &lt;html xmlns="http://www.w3.org/1999/xhtml" lang="$xcontext.locale.toLanguageTag()" xml:lang="$xcontext.locale"&gt;
```

Comment:
`xml:lang` is not supposed to use the same format ?

Reply:
https://www.w3.org/WAI/standards-guidelines/act/rules/5b7ae0/
You're right :+1: 
Addressed in https://github.com/xwiki/xwiki-platform/pull/2627/commits/2e4507395f94079f0c7735709ffb7b297681d82a